index.html

```diff
@@ -0,0 +1,225 @@
+              </div>
+            </div>
+          </div>
+    </div>
+    <footer>
+      <div class="footer-bg">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3 col-sm-3 col-12">
+                  <img src="images/logo.png" class="logo-footer">
+                </div>
+                <div class="col-md-3 col-sm-3 col-4">
+                  <div class="quick-links">
+                    <h5>Quick Links</h5>
+                    <div><a href="#">Home</a></div>
+                    <div><a href="#">Tracking</a></div>
+                    <div><a href="#">Contact</a></div>
+                    <div><a href="#">Sign Up</a></div>
+                    <div><a href="#">Login</a></div>
+                  </div>
+                </div>
+                <div class="col-md-4 col-sm-4 col-4">
+                  <div class="contact-footer">
+                    <h5>Contact Us</h5>
+                    <div class="cont-footer">
+                        <i class="fas fa-map-marker-alt"></i> <p>
+                        Kodehauz
+                        No. 38 AfahaUqua Road Eket,
+                        <br>Akwa Ibom State
+                      </p>
+                    </div>
+                    <div class="cont-footer">
+                        <i class="fas fa-phone"></i>
+                        <p>
+                        08064786137,09830587
+                      </p>
+                    </div>
+                    <div class="cont-footer">
+                        <i class="fas fa-envelope"></i>
+                         <p>
+                        info@Kodehauz.com
+                      </p>
+                    </div>
+                   
+                  </div>
+                </div>
+                <div class="col-md-2 col-sm-6 col-4 ph-scre">
+                    <div class="">
+                      <h5>Get in Touch</h5>
+                      <div class="social-media-icon">
+                        <a href="#" class="fb-icon"><i class="fab fa-facebook-f"></i></a>
+                        <a href="#" class="twitter-icon"><i class="fab fa-twitter"></i></a>
+                        <a href="#" class="instagram-icon"><i class="fab fa-instagram"></i></i></a>
+                       
+                      </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+      </div>
+      <div class="footer-designer">
+          <p class="text-center">	&copy; Copyright 2019 | Kodehauz </p>
+      </div>
+    </footer>
+    <script src="js/script.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+        <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+  
+
+  </body>
+</html>
```

Comment:
It is always recommended to add an extra line at the end of a source code.

Reply:
Well, we learn every day,  I have learned this very one today